Composer/plugins/vacore/template/assistant.dialog

```diff
@@ -0,0 +1,118 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "$designer": {
```

Comment:
Do you want the $designer nodes in these templates?  I've seen where the id get reused when starting projects from templates and although nothing really cares about the dialog IDs in the $designer node, it appears strange when in text editing mode.